UKHomeOffice/cerberus-service
# COP-6820 Refactor task mapping on the frontend to the new back end json

## Description
Refactor task details page to use new json payload received from camunda
## To Test
- Pull and run natively
- Navigate to a target
- *It should render successfully*

## Developer Checklist

\* Required
